Products/ZenUI3/browser/resources/js/zenoss/itinfrastructure.js

```diff
@@ -1142,6 +1142,7 @@ Ext.onReady(function () {
         }),
         router: REMOTE,
         nodeName: 'Device',
```

Comment:
now selectRootOnLoad functionality works as expected and in this case we should prevent selectRootOnLoad in device trees, because trees are rendered one by one and on page init device tree is already rendered and select root node on load but other trees are not rendered yet and this breaks page selection functionality;
instead we select root node in device tree manually when everything is rendered.